### PR TITLE
Feature#21 create a view to show house information

### LINKED ISSUE #21: Create a view to show house information
# REQUIREMENTS
- Should be designed mobile first 

# DESIGN
image

### PR DESCRIPTION
**Pull request by brocha05**
# Description
What did you implemented/Why did you implemented this:
- Create views to show single home in mobile and desktop 
- Add Javascript to allow the user view any photo of the house 

  ## Screenshots 
image
image

### CODE REVIEW

File: app/assets/stylesheets/main.scss [hunk 1]
```diff
@@ -10,6 +10,10 @@
 @import 'components/notifications';
 @import 'views/home';
 @import 'views/sign_up';
+<<<<<<< HEAD
+@import 'views/home_show';
+=======
```
Comment (YilKanda) [inline app/assets/stylesheets/main.scss]: Check your code, conflict tags appear, which you should remove:

```
<<<<<<< HEAD
=======
```

File: app/assets/stylesheets/main.scss [hunk 2]
```diff
@@ -10,6 +10,10 @@
...
 @import 'views/homes_index';
 @import 'views/reservation_form';
 @import 'views/reservation';
+>>>>>>> master
```
Comment (YilKanda) [inline app/assets/stylesheets/main.scss]: Check your code, conflict tags appear, which you should remove.

File: app/views/homes/show.html.erb [hunk 3]
```diff
@@ -1,3 +1,99 @@
...
+      </div>
+    </div>
+    <div class="options-show">
+      <a href= <%= edit_home_path(@home) %>><i class="fa-solid fa-pen-to-square"></i></a>
+      <a href= <%= homes_path %>><i class="fa-solid fa-arrow-rotate-left"></i></a>
+      <%= link_to '<i class="fa-solid fa-trash-can"></i>'.html_safe, homes_delete_path(@home) %>
+    </div>
+  </div>
+  <script> photos(); photosMobile(); </script>
+=======
```
Comment (YilKanda) [inline app/views/homes/show.html.erb]: Check your code, conflict tags appear, which you should remove

File: app/views/homes/show.html.erb [hunk 4]
```diff
@@ -1,3 +1,99 @@
+<<<<<<< HEAD
```
Comment (YilKanda) [inline app/views/homes/show.html.erb]: Check your code, conflict tags appear, which you should remove.

File: app/views/homes/show.html.erb [hunk 5]
```diff
@@ -6,4 +102,5 @@
   <%= link_to 'Reservar', new_reservation_path(home: @home)%>
 
   <%= button_to "Eliminar", @home, method: :delete %>
+>>>>>>> master
```
Comment (YilKanda) [inline app/views/homes/show.html.erb]: Check your code, conflict tags appear, which you should remove.

**Review by YilKanda — CHANGES_REQUESTED** [review]

File: app/assets/stylesheets/views/_home.scss [hunk 6]
```diff
@@ -40,6 +40,7 @@ main {
           img {
             height: 100%;
             width: 100%;
+            object-fit: cover;
```
Comment (dannharo) [inline app/assets/stylesheets/views/_home.scss:42]: sort this

**Review by dannharo — COMMENTED** [review]
check the comments of the git conflicts

File: app/assets/stylesheets/main.scss (hunk 1 above)
Comment (brocha05) [reply app/assets/stylesheets/main.scss]: Changes made

File: app/assets/stylesheets/views/_home.scss (hunk 6 above)
Comment (brocha05) [reply app/assets/stylesheets/views/_home.scss:42]: Changes made

**Review by AlejandroCuevasZepeda — APPROVED** [review]
LGTM

**Review by MrCano369x — APPROVED** [review]
lbtm

**Review by YilKanda — APPROVED** [review]
LGTM!

**Pull request merged**